Enable trusty and revert kernelflinger workaround

avinashiitk opened the pull request:
Enable all trusty related kernel compile flags for android15 build. Also revert kernelflinger prior workaround.

Tracked-On: OAM-134102

sysopenci commented:
one of the dependent pr cannot be Rebase and Merged; please fix the issue and commit your changes once again to run EB.

sysopenci commented:
Improper Commit Message
Tracked on not found in commit message,
    make sure Tracked-On: Jira-ticket  is present.

sysopenci commented:
one of the dependent pr cannot be Rebase and Merged; please fix the issue and commit your changes once again to run EB.

avinashiitk committed:
Enable trusty and revert kernelflinger workaround

Enable all trusty related kernel compile flags for android15 build.
Also revert kernelflinger prior workaround.

Tracked-On: OAM-134102
Signed-off-by: Avinash Kumar <avinash3.kumar@intel.com>

sysopenci commented:
one of the dependent pr cannot be Rebase and Merged; please fix the issue and commit your changes once again to run EB.

bnkoppad reviewed (approved):
LGTM
